### DISCUSSION TITLE
Fix "Memory" Page State Persistence Option

### DISCUSSION THREAD

**Discussion by thenotandy** [dnnsoftware/Dnn.Platform#4562] [Ideas]
I've been tempted over the years to enable the memory option under page state persistence, but can't due to the many issues. I did  some searching and found: https://dnntracker.atlassian.net/browse/DNN-24851

Does anyone know if the suggested solution is still a possible fix? Especially since there is already a provider included that supports web farms?

**Comment by mitchelsellers** [comment]
Personally, I think that we shoudl not think about this and really remove the feature as a whole from the platform.

The performance gain is minimal, and the risk is incredibly high.
